### DISCUSSION TITLE
Good Sites to find Live Wallpapers - Public contribution list

### DISCUSSION THREAD

**Discussion by NovusOpus** [rocksdanister/lively#1852] [Q&A]
Like the title says, Let's share some good places to find some nice Live Wallpapers. Only **`SFW(Safe For Work)`** content please: 

Some of the sites I found with good variety of Live wallpapers are: 
[https://moewalls.com/](https://moewalls.com/)
[https://www.pexels.com](https://www.pexels.com/search/videos/loop/) - Just search for `loop` on the `videos` section.

If you know some place/s, please feel free to share. Thanks!

**Comment by serialtoon** [comment]
Thanks for the recommendations, ive been looking for a site like this and you provided two! Thanks!!

**Comment by ShaggyDaddyDan** [comment]
This one is decent

https://mylivewallpapers.com/

**Comment by rocksdanister** [comment]
https://www.reddit.com/r/LivelyWallpaper/comments/kfvc2o/how_to_download_wallpapers_from_lively_deviantart/

**Comment by SteamPoweredCat** [comment]
Biggest collection of live wallpapers - https://www.desktophut.com

**Comment by MrCaen** [comment]
Another great site I recommend for [live wallpapers](https://wallsflow.com/) is Wallsflow.com
